### ISSUE TITLE
video carousel broken when scrolling past the last image

### ISSUE THREAD

**Issue by piuccio** [guardian/frontend#15272]
# ISSUE

## Steps to Reproduce

Go to https://www.theguardian.com/uk and scroll to the video carousel

Click on the right yellow arrow

![screen shot 2016-12-09 at 10 10 04](https://cloud.githubusercontent.com/assets/680284/21045336/dcc50dc6-bdf7-11e6-9bee-21f7275e5462.png)

Next video will show up, keep clicking on the arrow until you get to the last video.

Once you reach the end, the arrow is still visible (is this a problem already?) and if you click on it it'll throw an error

```
Potentially unhandled rejection [1] TypeError: Cannot read property 'classList' of null
    at t.<anonymous> (https://assets.guim.co.uk/javascripts/bootstraps/enhanced/media/977f679b40a4116c7c958c71cc24383a/main.js:9:9735)
    at t.<anonymous> (https://assets.guim.co.uk/javascripts/bootstraps/4cb1ce9b767c33f525d028f1f05fc55c/commercial.js:2:888)
    at t.run (https://assets.guim.co.uk/javascripts/a5231b953e04739d8db991f986407634/app.js:28:15807)
    at t.flush (https://assets.guim.co.uk/javascripts/a5231b953e04739d8db991f986407634/app.js:28:15367)
    at t.runBatch (https://assets.guim.co.uk/javascripts/a5231b953e04739d8db991f986407634/app.js:28:15513)
    at https://assets.guim.co.uk/javascripts/a5231b953e04739d8db991f986407634/app.js:28:15218
    at n (https://assets.guim.co.uk/javascripts/a5231b953e04739d8db991f986407634/app.js:28:15996)
```

the video fades away as if you moved to the next one

![screen shot 2016-12-09 at 10 13 48](https://cloud.githubusercontent.com/assets/680284/21045427/3c3b3924-bdf8-11e6-8241-6edbbd5fa731.png)

and clicks on the the arrows (left or right) will throw exceptions while doing nothing

## Expected Results (include screenshots)

Either hide the right arrow when you reach the last video, loop back to the first one or stop throwing errors

## OS and Browser details

Probably all, it happens on my Chrome latest

**Comment by paperboyo** [comment]
👋
[shyly] @sammorrisdesign ^

**Comment by piuccio** [comment]
I believe this was fixed by #15388
